tests/integ/test_df_to_pandas.py

```diff
@@ -141,6 +141,28 @@ def test_to_pandas_precision_for_number_38_0(session):
     assert pdf["A"].min() == -9223372036854775808
 
 
+def test_to_pandas_precision_for_number_38_6_and_others(session):
```

Comment:
Let's rename this test to `test_to_pandas_precision_for_non_zero_scale`

Reply:
Sounds good.